AY2223S1-CS2103-W14-1/tp
# Implement Add Client Command

This PR adds the add client command to the application. Closes https://github.com/AY2223S1-CS2103-W14-1/tp/issues/34

I refactored out Name and Address from `model.property` to `model.fields`, as I figured they were common across both property and clients.

I added some testutil functions for the JUnit tests, to help test for clients. Though not fully done, will slowly add.

To test:
1. Test the `add -c` command.
2. Check `data/clientDirectory.json` to see the new client
Note: There is no list command yet, so we can't see the new clients being added
